test/functional/test_framework/test_node.py

```diff
@@ -549,11 +550,17 @@ def p2p(self):
         assert self.p2ps, self._node_msg("No p2p connection")
         return self.p2ps[0]
 
+    def num_connected_mininodes(self):
+        """Return number of p2ps that are connected to the node. Excludes test nodes."""
```

Comment:
I think this would be slightly more concise if you say what you *are* counting (rather than saying what you're excluding):

```suggestion
        """Return number of test framework p2p connections to the node."""
```